circle.yml

```diff
@@ -44,6 +44,8 @@ test:
     - rvm $MRI_VERSIONS,$JRUBY_VERSIONS --verbose do appraisal rails4-mysql2 rake rails
     - rvm $RAILS5_VERSIONS --verbose do appraisal rails5-postgres rake rails
     - rvm $RAILS5_VERSIONS --verbose do appraisal rails5-mysql2 rake rails
+    - rvm $MRI_VERSIONS,$JRUBY_VERSIONS --verbose do appraisal elasticsearch rake elasticsearch
```

Comment:
can't we make a simple rake command like ``rake contrib``? for Rails it's different because we're testing all framework combinations that are a lot (different DBMS etc...). Just to avoid that we have a separate command for each integration